presto-spi/src/main/java/com/facebook/presto/spi/type/JsonType.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.type;
+
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+
+/**
+ * The stack representation for JSON objects must have the keys in natural sorted order.
+ */
+public class JsonType
```

Comment:
Is this a copy of `com.facebook.presto.type.JsonType`?

Reply:
yes. I copied it to spi to be aligned with other types

Reply:
@findepi 
Somehow I need to merge them in away that ```json``` column will be of type ```com.facebook.presto.spi.type.JsonType```

Reply:
done by extending ```com.facebook.presto.type.JsonType``` from ```com.facebook.presto.spi.type.JsonType``` and setting it's static JSON attribute to be ```com.facebook.presto.spi.type.JsonType.JSON```